main.js

```diff
@@ -1,2 +1,63 @@
+/*Test if everything is loading into browser*/
 console.log("Test: website is loading...")
 
+const API_URL = 'https://restcountries.eu/rest/v2/name'
+async function getCountryData(countryName='') {
+   try {
+       /**/
+       const API_CALL = (API_URL + '/' + countryName)
+       console.log('Calling: ' + API_CALL)
+       const response = await axios.get(API_CALL)
```

Comment:
Wat ik zelf heel vervelend vind is dat ik heel veel `response.data[0]` moet herhalen. Wij developers zijn heel lui, dus is daarom de object spreader in het leven geroepen 😉 
```
const { name: countryName, population, subregion: sbuAreaName, ...otherProps } = response.data[0];
```

_Je hoeft de props niet perse te aliasen_